packages/antd/src/widgets/AltDateWidget/index.tsx

```diff
@@ -14,11 +14,18 @@ export default function AltDateWidget<
   T = any,
   S extends StrictRJSFSchema = RJSFSchema,
   F extends FormContextType = any,
->(props: WidgetProps<T, S, F>) {
-  const { autofocus, disabled, id, name, onBlur, onFocus, options, readonly, registry } = props;
+>({
+  autofocus = false,
+  disabled = false,
+  options = { yearsRange: [1900, new Date().getFullYear() + 2] },
+  readonly = false,
+  time = false,
+  ...props
+}: WidgetProps<T, S, F>) {
+  const { id, name, onBlur, onFocus, registry } = props;
   const { formContext, translateString } = registry;
   const { rowGutter = 24 } = formContext as GenericObjectType;
-  const { elements, handleChange, handleClear, handleSetNow } = useAltDateWidgetProps(props);
+  const { elements, handleChange, handleClear, handleSetNow } = useAltDateWidgetProps({ ...props, autofocus, options });
```

Comment:
This code is equivalent to what the code in `getWidget()` does.  You will want to remove the default assignment above
```suggestion
  const realOptions = { yearsRange: [1900, new Date().getFullYear() + 2], ...options };
  const { elements, handleChange, handleClear, handleSetNow } = useAltDateWidgetProps({ ...props, autofocus, realOptions });
```